docker/images/pinot/etc/jmx_prometheus_javaagent/configs/controller.yml

```diff
@@ -216,6 +216,12 @@ rules:
   labels:
     tableName: "$1"
     tableType: "$2"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ControllerMetrics\", name=\"pinot.controller.tableRebalanceInProgress.([^\\.]*?)_(OFFLINE|REALTIME)\"><>(\\w+)"
```

Comment:
Do we need that? Shouldn't this be caught by the catch all rule?

Reply:
Catch-all rule PR was just merged, so this shouldn't be needed anymore. I'll test it out.

Reply:
Yes, it's working.